webapp/publisher/snaps/views.py

```diff
@@ -202,23 +204,32 @@ def publisher_snap_metrics(snap_name):
     # to use 10, rather then latest
     default_track = helpers.get_default_track(snap_name)
 
+    featured = False
     annotations = {"name": "annotations", "series": [], "buckets": []}
 
     for category in details["categories"]["items"]:
         date = category["since"].split("T")[0]
-        if date not in annotations["buckets"]:
-            annotations["buckets"].append(date)
+        if category["name"] != "featured":
+            if date not in annotations["buckets"]:
+                annotations["buckets"].append(date)
 
-        index_of_date = annotations["buckets"].index(date)
+            index_of_date = annotations["buckets"].index(date)
 
-        single_series = {
-            "values": [0] * (len(annotations)),
-            "name": category["name"],
-        }
+            single_series = {
+                "values": [0] * (len(annotations)),
+                "name": category["name"],
+            }
 
-        single_series["values"][index_of_date] = 1
+            single_series["values"][index_of_date] = 1
 
-        annotations["series"].append(single_series)
+            annotations["series"].append(single_series)
+        else:
+            new_date = datetime.datetime.strptime(date, "%Y-%m-%d")
```

Comment:
I think we have some logic function to format date that is used somewhere (snap details?). Maybe we could reuse it.

Reply:
Good catch, thanks @bartaz